frontend/src/components/NotFound/styles/ButtonContainer.ts

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
```

Comment:
Why do we have empty button container? Also in future it would be great to have all page specific style under a single file called styles.ts or elements.ts instead of creating multiple files.

Reply:
@jyash97 writing whole styles in one single file do not make sense 

but if we go multiple files we can easily find 

although it just a pattern of writing styles

Reply:
@nidhi-tandon @ankitnayan what pattern are we following with the project?

Reply:
What pattern do we see at [Jaeger Components](https://github.com/jaegertracing/jaeger-ui/tree/master/packages/jaeger-ui/src/components), [Grafana Components](https://github.com/grafana/grafana/tree/main/packages/grafana-ui/src/components) and [Mattermost Components](https://github.com/mattermost/mattermost-webapp/tree/master/components)? 

We can look up to them for inspiration.

Reply:
As suggested by @jyash97 will be using in a single file only